src/autopas/selectors/tuningStrategy/BayesianClusterSearch.h

```diff
@@ -80,13 +81,15 @@ class BayesianClusterSearch : public TuningStrategyInterface {
       const std::set<TraversalOption> &allowedTraversalOptions = TraversalOption::getAllOptions(),
       const std::set<LoadEstimatorOption> &allowedLoadEstimatorOptions = LoadEstimatorOption::getAllOptions(),
       const std::set<DataLayoutOption> &allowedDataLayoutOptions = DataLayoutOption::getAllOptions(),
-      const std::set<Newton3Option> &allowedNewton3Options = Newton3Option::getAllOptions(), size_t maxEvidence = 10,
+      const std::set<Newton3Option> &allowedNewton3Options = Newton3Option::getAllOptions(),
+      const std::set<int> &allowedVerletRebuildFrequencies = std::set<int>({12, 24, 48}), size_t maxEvidence = 10,
```

Comment:
Is there a reason that this is not a `NumberSet`?

Reply:
At one point it didn't work with Numberset and I couldn't find the reason.

Reply:
What line of code did you try and what was the error?

Reply:
I meant, it didn't compile at all.